src/ILLink.Shared/ILLink.LinkAttributes.xsd

```diff
@@ -0,0 +1,161 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xsd:schema attributeFormDefault="unqualified"
+		elementFormDefault="unqualified"
+		xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+
+	<xsd:complexType name="typechild">
```

Comment:
NIt: `typechild` doesn't sound right as a name - I would probably call it `membertype` or something like that. "member" in the type system means basically everything (type, method, field, proprety, ...). "child" has no meaning in type system.